src/app/docs/selection-box/selection-box-docs.component.html

```diff
@@ -0,0 +1,269 @@
+<sky-docs-demo-page
+  moduleName="SkySelectionBoxModule"
+  moduleSourceCodePath="src/app/public/modules/selection-box/"
+  pageTitle="Selection box"
+>
+  <sky-docs-demo-page-summary>
+    <!-- Page summary goes here -->
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo>
+    <!-- Demo goes here -->
+  </sky-docs-demo>
+
+  <sky-docs-design-guidelines>
+    
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+    
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p>
+          Use selection boxes when users must make a selection or answer a standalone question before proceeding, usually as part of a setup or configuration task. In these scenarios, the selection boxes should represent the only decision users need to make on a page.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use selection boxes for choices made during onboarding or other one-time processes."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/selection-box/selection-box-usage-1-modern.png"
```

Comment:
A couple minor things with this image. Given that the image is based on Tim's prototypes, I ran these suggestions by him already...

First, we have a grammatical mismatch between the question at the top of the image and 4 of the 5 options. The only option that grammatically fits as an answer to the question is `Religious education`. It's the only one where if you constructed the answer "I am responsible for [selection]," the resulting sentence makes sense. That's largely immaterial for demonstration purposes, but it still seems like we shouldn't encourage this sort of mismatch in our examples. I don't think it will cause enough confusion to prevent users from completing the form, but it feels a bit off, and I think users are going to pick up on that. I think we can fix that with a tiny tweak. If we just tweak the header to `What are your responsibilities?`, the resulting "My responsibilities are [selection]" sentence doesn't quite have the same disconnect. Still not perfect, but much better.

I'd change `Pick any` to `Select all` to use consistent terminology. This is super-trivial, but I'm seeing lots of questions about consistency in our UX content. We generally use `select` and sometimes `choose`, so I'd avoid adding a third alternative in `pick`. And in keeping with our discussion about filter options to select all items, I'd go with `all` over `any` because it's slightly more clear and removes any ambiguity.

Is it just me, or is the `Next` label smaller than it should be? I could be off base, but it seems too small.

Reply:
I've updated the terminology in the example image.

The marquee button is following modern theme specs, 16px bold font.